prompts.go

```diff
@@ -39,6 +42,10 @@ func PromptEnvDetails() bool {
 		migrationReq.Auth = TextInput("The environment variable 'HARNESS_MIGRATOR_AUTH' is not set. What is the api key?")
 	}
 
+	PromptUrlNG()
```

Comment:
These should not be here. We have the capability to create projects from the CLI. These prompts are not required in such cases.

Reply:
We are also planning to add support for providing a summary of accounts and apps. These prompts are not required in such cases